src/pages/Admin/Admin.tsx

```diff
@@ -364,16 +348,17 @@ const VenueInfoComponent: React.FC<VenueInfoComponentProps> = ({
                 Edit Room
               </Link>
             )}
-            <button
-              className="btn btn-primary"
-              onClick={() => {
-                history.push(`${match.url}/events`);
-                setShowCreateEventModal(true);
-              }}
-              style={{ marginBottom: 10, width: "100%" }}
-            >
-              Create an Event
-            </button>
+
+            {canBeDeleted(venue) && (
+              <button
+                role="link"
+                className="btn btn-block btn-danger"
+                onClick={showDeleteModal}
+              >
+                Delete space
+              </button>
+            )}
```

Comment:
Where is coming from? I though we already had the delete button in place

Reply:
This change comes from merge-commit. I don't know why is it highlighted here.

Reply:
Can you confirm that there are no `Delete space` button?

Reply:
I don't know if it's a merge error or I accidentally did something wrong but after merging recent changes from staging there really were two "Delete space" buttons. And unused code I remove became used again:) So I think somebody was working on that "Delete space" button and I interfered. Anyway, fixed in https://github.com/sparkletown/sparkle/pull/1382/commits/f8ad957794093233f6547c039f4063c27482d813